zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedDiskChecker.scala

```diff
@@ -40,7 +40,7 @@ object DeletedDiskChecker {
                     ee.getCause.getMessage.contains("Compute Engine API has not been used") || ee.getCause.getMessage
                       .contains("This API method requires billing to be enabled")
                   )
-                    if (isDryRun) F.unit else dbReader.updateDiskStatus(disk.id)
+                    logger.info(s"failed to get ${disk.diskName} due to ${ee.getCause.getMessage}")
```

Comment:
just occured to me, it'd be nice to add some metrics for this..just so we have a sense how things are going in the wild...

maybe we can chat abt it next mobbing or sth?

Reply:
https://github.com/broadinstitute/leonardo-cron-jobs/blob/87b615eebf93143bf94c27466cbe9bd0687a888b/resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DataprocWorkerChecker.scala#L72 is an example

Reply:
That's a great idea. If you get a few minutes, I would love to talk with you about whatever setup is necessary to get that working.